mesheryctl/pkg/utils/testing.go

```diff
@@ -127,6 +139,21 @@ func SetupContextEnv(t *testing.T) {
 func StartMockery(t *testing.T) {
 	// activate http mocking
 	httpmock.Activate()
+
+	// get current directory
+	_, filename, _, ok := runtime.Caller(0)
+	if !ok {
+		t.Fatal("problems recovering caller information")
+	}
+	currDir := filepath.Dir(filename)
+	fixturesDir := filepath.Join(currDir, "fixtures")
+
+	apiResponse := NewGoldenFile(t, "validate.version.github.golden", fixturesDir).Load()
```

Comment:
validate.version.github.golden file has so many entries. Do we want to load and compare all the entries in API response?

Reply:
Ahh, This has been set up for something that's being called on the backend, where meshery tries to validate the config each time we try to fetch the basemesheryurl each time from config.